src/fastoad/io/configuration/configuration.py

```diff
@@ -249,6 +252,62 @@ def set_optimization_definition(self, optimization_definition: Dict):
         subpart = {"optimization": subpart}
         self._data.update(subpart)
 
+    def make_local(self, new_folder_path: Union[str, PathLike], copy_models: bool = False):
+        """
+        Modify the current configurator so that all input and output files will be in
+        the indicated folder.
+
+        :param new_folder_path:
+        :return:
+        """
+        new_folder_path = Path(new_folder_path)
+
+        self._data[KEY_INPUT_FILE] = self._make_path_local(self.input_file_path, new_folder_path)
+        self._data[KEY_OUTPUT_FILE] = self._make_path_local(self.output_file_path, new_folder_path)
+        if copy_models:
+            new_model_folders = []
+            for i, module_folder_path in enumerate(self._get_module_folder_paths()):
+                if module_folder_path.is_dir():
+                    new_model_folders.append(
+                        self._make_path_local(
+                            module_folder_path, new_folder_path, local_path=f"models_{i}"
+                        )
+                    )
+            self._data[KEY_FOLDERS] = new_model_folders
+        else:
+            # Make model folder paths absolutes, since configuration file will be moved.
+            # _get_module_folder_paths() already does the job when the configuration file
+            # is at its original location.
+            self._data[KEY_FOLDERS] = [
+                module_folder_path.as_posix()
+                for module_folder_path in self._get_module_folder_paths()
+            ]
+
+        self._make_options_local(self._data[KEY_MODEL], new_folder_path)
```

Comment:
Shouldn't this function be called on `self._data[KEY_MODEL_OPTIONS]` as well? The sellar problem does contain a input file path in the model options. When running the test, the path is edited but the file is not copied locally...

Reply:
Indeed!!! Fixed in 9b086a0924a9763be7519f329ed697f7c0b5f48d
Looks like I wanted to check that at some point... and completely lost track of it in the process 😅 

Ideally, paths in `model_options` would have been converted in local paths wrt to the components that actually use the option (like the "normal" options), but that was too long a shot. We stick to the "hierarchy" inside `model_options` definition, with the removal of glob wildcards (* and ?) that would pollute the usage of generated paths.
After all, the rationale for making these local paths is to ensure different files with the same name won't be mixed up in the same location, which should be achieved by the present solution.